packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -843,6 +853,22 @@ const useQuickActions = () => {
         disabled: !isReady || !hasAudio,
         ...actionMenuProps,
       },
+      videoInput && {
+        Icon: videoEffect === 'blur' ? BackgroundBlur : BackgroundBlurOff,
+        label:
+          videoEffect === 'blur'
+            ? __('Disable Background Blur', 'web-stories')
+            : __('Enable Background Blur', 'web-stories'),
+        onClick: () => {
+          trackEvent('media_recording_background_blur', {
+            status: videoEffect === 'blur' ? 'off' : 'on',
```

Comment:
Thinking: if we ever add more blur steps (e.g. strong/medium/light blur), then it might be easier to pass a number here, e.g.

```suggestion
            value: videoEffect === 'blur' ? 0 : 100,
```

Reply:
Do we want the historical data to be meaningful after the changes? If we do 0-100% of a scale, then the old data might be skewed if we change the scale. If we track blur px values (0px blur, 3px blur, 6px blur) then the old data will be still valid.

Reply:
Good point with the scale 👍  Tracking the px values sounds more useful in that case